### PR TITLE
Added support for Box as storage engine

### PR DESCRIPTION
**Pull request by ninjabong**
I use Box, so I added support for it which was, thankfully, trivial.

### CODE REVIEW

**Comment by lra** [comment]
Tests are failing ;)

**Comment by ninjabong** [comment]
Yeah not sure why that was barfing, but working now 8)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## WIP
 
+- Added support for Box to store you files (via @ninjabong)
```
Comment (lra) [inline CHANGELOG.md]: typo

**Comment by lra** [comment]
Awesome! thanks, merged =)
Will bump a new version today.

**Pull request merged**